gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -76,29 +77,38 @@ function GM:PlayerBindPress(ply, bindName, pressed)
             return TBHUD:UseFocused()
         end
 
-        local tr = util.TraceLine({
-            start = ply:GetShootPos(),
-            endpos = ply:GetShootPos() + ply:GetAimVector() * 100,
-            filter = ply,
-            mask = MASK_SHOT,
-        })
-
-        if not tr.Hit or not IsValid(tr.Entity) then
-            return
-        end
-
+        -- Find out if a marker is focussed otherwise check normal use
         local isClientOnly = false
-        if isfunction(tr.Entity.ClientUse) then
-            isClientOnly = tr.Entity:ClientUse()
+        local useEnt = markerVision.GetFocussedEntity()
+        local isRemote = IsValid(useEnt)
+        if not isRemote then
+            local tr = util.TraceLine({
+                start = ply:GetShootPos(),
+                endpos = ply:GetShootPos() + ply:GetAimVector() * 100,
+                filter = ply,
+                mask = MASK_SHOT,
+            })
+
+            useEnt = tr.Entity
+            if not tr.Hit or not IsValid(useEnt) then
+                return
+            end
+
+            if isfunction(useEnt.ClientUse) then
+                isClientOnly = useEnt:ClientUse()
+            end
+        elseif isfunction(useEnt.RemoteUse) then
+            isClientOnly = useEnt:RemoteUse(ply)
```

Comment:
This would prioritize focused marker ents over "normal" use scenarios even if the marker doesn't have RemoteUse functionality, right?

From a gameplay perspective i'm not sure if i like this or not.
It is probably good because it is always the same no matter what marker ent happens to be focused.

Reply:
Yes, the current implementation of markers are drawn over everything and it even gets bigger on focus, so I think its clear, that nothing else should get triggered.
Even if the marker has no implementation clienside, it still might be that there is an implementation serverside.